data/Creek_2018/data.csv

```diff
@@ -0,0 +1,4 @@
+﻿taxon_name,Leaf Psi50,Leaf Psi12,Leaf Psi88,Stem Psi50,Stem Psi12,Stem Psi88,Root Psi50,Root Psi12,Root Psi88,Leaf Psi50-Stem Psi50,Root Psi50-Stem Psi50,PsiA80,Psigs80,PsiE80,PsiTLP,PsiTLP_SE,Psigs80-leaf Psi50,Psigs80-stem Psi50,Psigs80-root Psi50,Psigs80-leaf Psi12,Psigs80-stem Psi12,Psigs80-root Psi12
+Acacia aneura,-5.84,-2.59,-10,-8.46,-6.54,-10.04,-8.16,-5.3,-10.86,2.62,0.3,-4.96,-2.63,-2.07,-1.66,0.06,3.21,5.83,5.53,0.04,3.91,2.67
```

Comment:
I fixed it, but there were weird "non space" characters between words for the taxon names